com.unity.multiplayer.mlapi/Runtime/Core/RPCQueue/RPCQueueProcessing.cs

```diff
@@ -22,6 +23,9 @@ internal class RPCQueueProcessing
         static ProfilerMarker s_MLAPIRPCQueueProcess = new ProfilerMarker("MLAPIRPCQueueProcess");
         static ProfilerMarker s_MLAPIRPCQueueSend = new ProfilerMarker("MLAPIRPCQueueSend");
 #endif
+        // Batcher object used to manage the RPC batching on the send side
+        private MessageBatcher batcher = new MessageBatcher();
+        private int BatchThreshold = 1000;
```

Comment:
we will probably move message batcher somewhere else in the future but for now, I think I'm OK with this.